src/adagrams.js

```diff
@@ -1,15 +1,95 @@
+import { letterPool, scoreChart } from 'constants';
+
+// helper function to create letter pool array
+export const createLetterPool = () => {
+  const letterPoolArray = [];
+
+  for (const letter in letterPool) {
+    for (let i = 0; i < letterPool[letter]; ++i) {
+      letterPoolArray.push(letter);
+    }
```

Comment:
This isn't efficient for space, but as an example of things we can do with the spread operator, we could replace the inner for-loop with something like:
```
letterPoolArray.push(...new Array(letterPool[letter]).fill(letter));
```